pom.xml

```diff
@@ -178,10 +178,8 @@
         </dependency>
         <dependency>
             <groupId>org.springframework.cloud</groupId>
-            <artifactId>spring-cloud-stream</artifactId>
-            <type>test-jar</type>
+            <artifactId>spring-cloud-stream-test-binder</artifactId>
```

Comment:
OK saw the migration instructions from [
Spring Cloud Stream
](https://spring.io/blog/2022/12/02/spring-cloud-2022-0-0-release-candidate-3-codename-kilburn-has-been-released)

> The test binder in Spring Cloud Stream moved to a separate artifact from the core Spring Cloud Stream module. Before this release, the test binder was part of the core module, and users used it using the test-binder classifier, which was inconvenient. With this release, the test binder is now moved and available through spring-cloud-stream-test-binder module. You can add it as a dependency in your application and get all the previous test binder components.